requirements-vllm-cuda.txt

```diff
@@ -2,4 +2,4 @@
 # Dependencies for installing vLLM on CUDA
 
 # vLLM only supports Linux platform (including WSL)
-vllm==0.7.3 ; sys_platform == 'linux' and platform_machine == 'x86_64'
+vllm>=0.8.0,<0.9.0 ; sys_platform == 'linux' and platform_machine == 'x86_64'
```

Comment:
I would prefer if you would remove the upper version cap and instead move the constraint into `constraints-dev.txt`. If you keep the upper version cap, then we'll have to request an InstructLab update every time we want to test a new version of vLLM.

Reply:
Sure, makes sense. I like that better. I'll remove that upper cap here and add to `constraints-dev.txt`

Reply:
Updated.